app/src/androidTest/java/org/mozilla/fenix/ui/NoNetworkAccessStartupTests.kt

```diff
@@ -0,0 +1,121 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.ui
+
+import androidx.core.net.toUri
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.IdlingRegistry
+import org.junit.After
+import org.junit.Test
+import org.mozilla.fenix.R
+import org.mozilla.fenix.helpers.HomeActivityTestRule
+import org.mozilla.fenix.helpers.TestHelper.packageName
+import org.mozilla.fenix.helpers.TestHelper.setNetworkEnabled
+import org.mozilla.fenix.helpers.TestHelper.verifyUrl
+import org.mozilla.fenix.helpers.idlingresource.NetworkConnectionIdlingResource
+import org.mozilla.fenix.ui.robots.browserScreen
+import org.mozilla.fenix.ui.robots.homeScreen
+import org.mozilla.fenix.ui.robots.navigationToolbar
+
+/**
+ * Tests to verify some main UI flows with Network connection off
+ *
+ */
+
+class NoNetworkAccessStartupTests {
+    private val activityTestRule = HomeActivityTestRule()
```

Comment:
Not sure if it's what you want here but I added `HomeActivityTestRule(skipOnboarding = true)` if you want to avoid having to do:
```kotlin
homeScreen {
}.dismissOnboarding()
```

It sets the preference before the app starts so that it thinks it's already shown the onboarding screen.

Reply:
I just wanted to do some basic navigation after startup.